core/src/test/resources/dumped-smart-testing.yml

```diff
@@ -0,0 +1,9 @@
+!!org.arquillian.smart.testing.configuration.Configuration
```

Comment:
the directory ` core/src/test/resources` is getting to be chaotic - move the config files to separated subdirectory.